src/libraries/pkg/Microsoft.NETCore.Platforms/runtimeGroups.props

```diff
@@ -15,7 +15,7 @@
 
     <RuntimeGroup Include="alpine">
       <Parent>linux-musl</Parent>
-      <Architectures>x64</Architectures>
+      <Architectures>x64;arm64</Architectures>
```

Comment:
Do all these versions support arm64?  If not we could create a separate group for arm64.

Reply:
Yes, all the version back to 3.6 support arm64.